packages/fxa-settings/src/models/integrations/oauth-web-integration.ts

```diff
@@ -422,21 +426,10 @@ export class OAuthWebIntegration extends BaseIntegration {
       if (claims.sub !== account.uid) {
         throw new OAuthError('PROMPT_NONE_DIFFERENT_USER_SIGNED_IN');
       }
-    } else {
-      if (!requestedEmail) {
-        // yeah yeah, it's a bit strange to look at `email`
-        // and then say `login_hint` is missing. `login_hint`
-        // is the OIDC spec compliant name, we supported `email` first
-        // and don't want to break backwards compatibility.
-        // `login_hint` is copied to the `email` field if no `email`
-        // is specified. If neither is available, throw an error
-        // about `login_hint` since it's spec compliant.
-        throw new OAuthError('login_hint');
-      }
```

Comment:
content server UI did not require an email or login_hint when there is no idTokenHint.

Reply:
I swear this comment was transferred over from content-server, but I don't see it now. 👍